src-tauri/src/development/mod.rs

```diff
@@ -0,0 +1,81 @@
+use crate::github::{
+    pull_requests::{check_github_api, download_zip_into_memory, get_launcher_download_link},
+    CommitInfo,
+};
+
+#[tauri::command]
+pub async fn install_git_main(game_install_path: &str) -> Result<String, String> {
+    let url = "https://api.github.com/repos/R2Northstar/NorthstarLauncher/commits";
```

Comment:
Shouldn't this be in the constants package?

Reply:
Done with 023fee1ea15a6b44cea4fe0aa878a45f95b0d385